src/tensor_product.jl

```diff
@@ -1,6 +1,14 @@
 # This files defines an interface for the tensor product of two axes
 # https://en.wikipedia.org/wiki/Tensor_product
 
+# ==================================  misc  ================================================
+is_offset_axis(a::AbstractUnitRange) = isone(first(a))
```

Comment:
I think this definition is backwards and should instead be:
```julia
is_offset_axis(a::AbstractUnitRange) = !isone(first(a))
```
i.e. it is an "offset axis" if it doesn't start at one.